src/test/rustdoc/doc-cfg-target-feature.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// only-x86_64
+// compile-flags:--test
+// should-fail
+
+#![feature(cfg_target_feature)]
+
+// #49723: rustdoc didn't add target features when extracting or running doctests
+
+/// Foo
+///
+/// # Examples
+///
+/// ```
+/// assert!(false);
+/// ```
+#[cfg(target_feature = "sse")]
```

Comment:
If this doctest fails, is because foo was not part of the binary, so... "sse" is not set for some reason, which is very weird. It is set for x86_64 always. cc @alexcrichton

Reply:
If you run the test locally (i.e. `x.py test --stage 1 src/test/rustdoc --test-args "target-feature"`) then it will work properly, implying there's something off with the Travis setup.

Reply:
Aha right yes, you can tag this as `// no-system-llvm`